modules/ROOT/pages/android_app/publishing.adoc

```diff
@@ -266,98 +251,98 @@ As part of creating your Google Play Publisher account you will have to create s
 === Create a Google Play Publisher Account
 
 Start at Google’s http://developer.android.com/distribute/googleplay/start.html[Get Started With Publishing] page.
-Have a credit card ready, because it costs $25.
-If you already have a Google account, it is usually better to create a separate new account just for publishing apps to the Google Play Store.
+Have a credit card ready, because it costs about $25. If you already have a Google account, it is usually better to create a separate new account just for publishing apps to the Google Play Store.
 
-Google’s process for uploading apps is fairly streamlined, and the most time-consuming task is creating all the required graphics.
-After registering, you’ll see the welcome screen for the Google Dev Console.
-Click *Publish an Android app on Google Play*.
+Google’s process for uploading apps is fairly streamlined, and the most time-consuming task is creating all the required graphics. After registering, you’ll see the welcome screen for the Google Dev Console. Click btn:[Publish an Android app on Google Play].
 
-image:branded_android_app/android_custom_6.png[image]
+image::android_app/publishing/android_publish.png[Android App Publish, width=450]
 
-This opens the *Add New Application* screen.
-Click the *Prepare Store Listing* button. (Note that as you navigate the various screens, you can click the Save Draft button to preserve your changes.)
+This opens the menu:Add New Application[] screen. Click the btn:[Prepare Store Listing] button. Note that as you navigate the various screens, you can click the btn:[Save Draft] button to preserve your changes.
 
-image:branded_android_app/android_custom_7.png[image]
+image::android_app/publishing/android_new_application.png[Android New Application, width=450]
 
 On the next screen, enter your product description.
 
-image:branded_android_app/android_custom_8.png[image]
+image::android_app/publishing/android_product_description.png[Android Product Description, width=450]
 
-Then you’ll have to upload a batch of graphics in various sizes for the *Graphic Assets* section, like these images for a smartphone and seven-inch tablet.
-You are required to upload at least two images.
+Then you’ll have to upload a batch of graphics in various sizes for the *Graphic Assets* section, like these images for a smartphone and seven-inch tablet. You are required to upload at least two images.
 
-image:branded_android_app/android_custom_9.png[image]
+image::android_app/publishing/android_screenshots.png[Android Screenshots, width=450]
 
 You must also upload a 512x512-pixel logo, and a 1024x500 banner.
 
-image:branded_android_app/android_custom_10.png[image]
+image::android_app/publishing/android_logo_and_banner.png[Android Screenshots, width=450]
```

Comment:
Should the text say "Android Screenshots", or something different?

Reply:
fixed